File: v01/Examples/animationWallPusher/tests/files/lib/java/Robot.java
package algorea;

public class Robot
{
   public static void __deplace(int dL, int dC, int dep)
   {
      System.out.println("");
      System.out.println("CMD");
      System.out.println("move");
      System.out.println("3");
      System.out.println(dL);
      System.out.println(dC);
      System.out.println(dep);
      System.out.println("");
   }

   public static void haut(int dep)
   {
      __deplace(-1, 0, dep);
   }
   public static void bas(int dep)
   {
      __deplace(1, 0, dep);
   }
   public static void gauche(int dep)
   {
      __deplace(0, -1, dep);
   }
   public static void droite(int dep)
   {
      __deplace(0, 1, dep);
   }
}

File: v01/Examples/animationWallPusher/tests/files/lib/ocaml/Robot.ml
open Printf

let read_int() = Scanf.scanf " %d" (fun x -> x)

let __deplace dL dC dep =
   printf("\n");
   printf("CMD\n");
   printf("move\n");
   printf("3\n");
   printf("%d\n") dL;
   printf("%d\n") dC;
   printf("%d\n") dep;
   printf("\n")

let haut dep =
   __deplace (-1) 0 dep

let bas dep =
   __deplace 1 0 dep

let gauche dep =
   __deplace 0 (-1) dep

let droite dep =
   __deplace 0 1 dep
